server/handlers.go

```diff
@@ -151,6 +152,7 @@ type discovery struct {
 	Auth          string   `json:"authorization_endpoint"`
 	Token         string   `json:"token_endpoint"`
 	Keys          string   `json:"jwks_uri"`
+	UserInfo      string   `json:"userinfo_endpoint"`
```

Comment:
Could you expand this test case for the discovery json, please? https://github.com/dexidp/dex/blob/8959dc42750e1f574db74eec9f0fcc61b39b83ea/server/server_test.go#L150